### ISSUE TITLE
8416 - 508 Audit - Defect 1 - Critical: Incorrect value for  Step 4 "your full name" field

### ISSUE THREAD

**Issue by maggiewachs** [department-of-veterans-affairs/va.gov-team#130429]
On the Submit Medical Expenses To Support A Pension Or DIC Claim | Veterans Affairs screen (You are on Step 4 of 4: Review application), when tab is used to move the focus to Your full name (*Required), "… Veteran Test Statement of truth: I confirm that the identifying information …"is read by assistive technology—this does not convey the correct value in the field. (New: 01/16/2026)

**Comment by maggiewachs** [comment]
Unable to reproduce with VoiceOver, so we should have someone recheck with JAWS. Also worth noting that our implementation exactly matches the ["statement of truth" design system component](https://design.va.gov/components/form/statement-of-truth).
